core/src/main/java/de/danoeh/antennapod/core/feed/SubscriptionFilter.java

```diff
@@ -0,0 +1,127 @@
+package de.danoeh.antennapod.core.feed;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SubscriptionFilter {
+    public static final String STRING_ENABLE = "Enable";
+    public static final String STRING_DISABLE = "Disable";
+    public static final String STRING_ALWAYS = "Always";
+    public static final String STRING_NEVER = "Never";
+    public static final String STRING_GLOBAL_DEFAULT = "Global default";
+    public static final String STRING_NO_FILTER = "No filter";
+
+    public enum AutoDownload{
+        ENABLE(STRING_ENABLE),
+        DISABLE(STRING_DISABLE),
+        NO_FILTER(STRING_NO_FILTER);
```

Comment:
Instead of having those strings, you can also just use `AutoDownload.ENABLE.name()` to get a string representation. This would make this class a lot shorter.